target/linux/ath79/dts/ar9342_ubnt_wa.dtsi

```diff
@@ -11,8 +11,6 @@
 
 	keys {
 		compatible = "gpio-keys";
-		#address-cells = <1>;
```

Comment:
I think this slipped from editing previous commit. It would be nice to fix it in previous commit by "git rebase -i".